dali/test/python/test_dali_cpu_only.py

```diff
@@ -633,5 +633,8 @@ def test_sequence_reader_cpu():
 
 def test_affine_transforms_cpu():
     check_no_input(fn.translate_transform, offset=(2, 3))
+    check_no_input(fn.scale_transform, scale=(2, 3))
```

Comment:
I would slip it into separate test cases, but maybe it is an overkill...

Reply:
done